.mailmap

```diff
@@ -17,3 +17,4 @@ Steven Solie <steven@solie.ca> <ssolie@users.sourceforge.net>
 Thomas Beutlich <tc@tbeu.de>
 Viktor Szakats <commit@vsz.me> <vszakats@users.noreply.github.com>
 Zhongyuan Zhou <zhouzhongyuan@huawei.com>
+Donghee Na <donghee.na@python.org>
```

Comment:
While the entry looks good and seems to work well in practice, previously the lines in this file were sorted alphabetically. Could you move this line after `Ben Wagner` please?